### ISSUE TITLE
Idle timer (for freeze, self-kill)

### ISSUE THREAD

**Issue by eMPee584** [ranger/ranger#2101]
It would be useful to set up an idle timer for toggling `freeze_files` .. Maybe this can help for situations with automounted sshfs and huge terminal sessions with many ranger instances.. Thinking of that, a self-kill mode would also be helpful for temporary ranger panes in tmux..
Don't have the time to work on this atm but I would guess it takes less than two dozen lines of code..

**Comment by toonn** [comment]
Does ranger even load a directory that's already considered loaded? I'm not so sure this would deliver any improvement.

**Comment by eMPee584** [comment]
Yes it seems so, I often see a looping dir refresh.. especially in scenarios with autofs or symlinked svg documents on /dev/shm involved .. 😅💦

**Comment by toonn** [comment]
I think you're just seeing ranger not being finished loading something. It shouldn't be refreshing much without action on your part at all.

**Comment by eMPee584** [comment]
Ok a self kill idle-timer would still be useful for tmux sessions when f.e. using ranger as directory browser to execute shell commands tmux panes..
